test/test_nn.py

```diff
@@ -624,6 +624,17 @@ def test_Dropout3d(self):
         input = torch.Tensor(num_features, b, d, w, h)
         self._test_dropout(nn.Dropout3d, input)
 
+    def test_pad(self):
+        inputs = Variable(torch.randn(1, 3, 4, 4), requires_grad=True)
+        self.assertTrue(gradcheck(lambda x: F.pad(x, (1, 1, 1, 1)), (inputs,)))
```

Comment:
@apaszke it seems a bit weird to mix autograd and nn functions in test_autograd.py, is it ok?